src/core/main.go

```diff
@@ -102,11 +104,32 @@ func main() {
 	redisURL := os.Getenv("_REDIS_URL")
 	if len(redisURL) > 0 {
 		gob.Register(models.User{})
-		beego.BConfig.WebConfig.Session.SessionProvider = "redis"
-		beego.BConfig.WebConfig.Session.SessionProviderConfig = redisURL
+		if strings.HasPrefix(redisURL, "redis+sentinel://") {
```

Comment:
Extract such URL parsing logic into a helper function for easily understand and well code organized purposes?